package.json

```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "async": "^1.5.0",
     "bluebird": "^3.1.1",
-    "cloudant": "^1.4.0",
+    "cloudant": "^1.10.0-NOTICE",
```

Comment:
Should we use `@cloudant/cloudant`? And why are we using a version tagged with `-NOTICE`?

Reply:
@virkt25 I copied from the PR that fixed the installation problem and also tried on local "^1.10.0-NOTICE" works.
`@cloudant/cloudant` may contain breaking change that we are not compatible with. If time allows we can try upgrading it in `loopback-connector-cloudant@2.x`, but for `loopback-connector-cloudant@1.x` I would rather stay with 1.x...

Reply:
@virkt25 see this PR https://github.com/strongloop/loopback-connector-cloudant/pull/186